Catalog doc (#95)
miamtech/mealz-documentation: docs/android/features/catalog/walkthrough.md

The Catalog is all of our Recipes, with additional features & options to filter, search, add dietary restrictions, & a whole lot more. 

<TimeIcon titleText="Time to read:" timeText="30 minutes" /><br />
<TimeIcon titleText="Time for base implementation:" timeText="2 hours" /><br />
<TimeIcon titleText="Time for custom implementation:" timeText="8 hours" /><br />
<TimeIcon titleText="Time for full customization:" timeText="2.5 weeks" /><br />

## Prerequisites
<Prerequisites />

## Ingredients

> There is two diffrente way to add this component to your projet : 
>     - basic implementation : an all in one compoment that embed naviagation and all needed other component
>     - custom implementation : were you can handle naviagtion by yourself and link component that you need 
> 

Reviewer: ```suggestion
There are two diffrent ways to add this component to your project : 
    - basic implementation : an all-inone component that embed navigation and all needed other components
    - custom implementation : were you can handle navigation by yourself and link components that you need 

```
